src/template.jl

```diff
@@ -217,8 +217,8 @@ function prompt(::Type{Template}, ::Type, ::Val{:host})
 end
 
 function prompt(::Type{Template}, ::Type, ::Val{:julia})
-    versions = map(format_version, [VERSION; map(v -> VersionNumber(1, v), 0:5)])
-    push!(sort!(unique!(versions)), "Other")
+    versions = map(format_version, map(v -> VersionNumber(1, v), 0:VERSION.minor))
```

Comment:
This makes me want to implement ranges for `VersionNumber` (e.g. `v"1.0.0":v"0.1":VERSION`)
```suggestion
    versions = map(format_version, VersionNumber.(1, 0:VERSION.minor))
```

Reply:
Grrr I like `map` so much more than dot. But oh well 🙃

Reply:
`map` has it's place but sometimes dot is much faster and easier to read